rest-server/core/src/main/scala/com/actionml/core/store/Store.scala

```diff
@@ -19,11 +19,12 @@ package com.actionml.core.store
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.reflect.ClassTag
+import scala.reflect.runtime.universe._
 
 
 trait Store {
   def dbName: String
-  def createDao[T](collectionName: String)(implicit ct: ClassTag[T]): DAO[T]
+  def createDao[T: TypeTag](collectionName: String)(implicit ct: ClassTag[T]): DAO[T]
```

Comment:
Why change from just [T] to a TypeTag?

Reply:
because we need meta information - annotations